smoke-tests/smoke-test-cjs.cjs

```diff
@@ -0,0 +1,24 @@
+// @ts-check
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
+
+const assert = require('node:assert');
+const concurrently = require('../index.js');
+
+(async () => {
+  try {
+    // Assert the functions loaded by checking their names load and types are correct
+    assert.strictEqual(
+      typeof concurrently === 'function',
+      true,
+      'Expected default to be function',
+    );
+
+    console.info('Imported cjs successfully');
+  } catch (error) {
+    console.error(error);
+    console.debug(error.stack);
```

Comment:
Node logs the stacktrace by default, doesn't bun do the same? E.g. if you make the assertion fail, won't it print the stack trace twice?